### ISSUE TITLE
Looking for new mainainer

### ISSUE THREAD

**Issue by dtandersen** [factoriotools/factorio-docker#229]
Hey guys, I haven't played Factorio for like 2 years. But I really like the game so I kept maintaining this.

But now I feel I'm holding back progress, so I think it's time to hand off. The question is who or whom?

**Comment by DBendit** [comment]
@Fank seems to have been pretty active on here, so if there's any sort of nomination process, I think that's where my vote goes.

**Comment by dtandersen** [comment]
I've added deveth0 and  danstis for now. 

Who was it that's trying to get the auto mod updating done?

**Comment by Fank** [comment]
Im only here for docker knowledge and keep auto Update working.
Havnt played factorio for 1+ year.

**Comment by danstis** [comment]
> Who was it that's trying to get the auto mod updating done?

I think that is @patschi

**Comment by SuperSandro2000** [comment]
I don't really play Multiplayer Factorio cause the latency is to high for me but I love modded in Singleplayer or as the host. I am familiar with bash and docker and almost all my projects contain either one. I would have enough time to keep the project and existing features running but I don't really have time to develop anything new and fancy. So if you really need another man just in case feel free to add me or mention me. I usually get around answering within a day.

**Comment by patschi** [comment]
> > Who was it that's trying to get the auto mod updating done?
> 
> I think that is @patschi

Not fully correct - the first PR with the major work is done by @ALovedOne in #222. I've just done a bit of rework and squashed a few bugs with a later PR in #224.

**Comment by Fank** [comment]
I totally agree with @mgh87 the only problem i see is that the name could change.

**Comment by wellczech** [comment]
Factorio Assembly Machine

**Comment by SuperSandro2000** [comment]
Assembly required

**Comment by danstis** [comment]
"Humans Need Not Apply"

**Comment by danstis** [comment]
"The Factory Must Grow"

**Comment by patschi** [comment]
What about something simple like "FactorioTools"?

**Comment by mgh87** [comment]
FactorioTools is available on Github+Dockerhub so that's a start :smiley_cat: 
There are currently 16 different versions of this docker image available on dockerhub and 151 forks of this repository.
If I find the time I would support you guys to create a CI/CD pipleline for easier deployment of new builds.
I'm also currently not really playing and don't host a server so for working on the actual docker images I won't be a lot of help.

**Comment by m-chandler** [comment]
I'd be keen to get more involved. Background is mostly AWS and web dev, but familiar with Docker & Linux etc. I'll pitch in and help out as needed.

**Comment by SuperSandro2000** [comment]
Maybe we should switch to an organisation on docker hub? Maybe call it factoriotools?

**Comment by Fank** [comment]
Sorry for my late statement.

What will happen the next days:
@SuperSandro2000 added the travis intigration for factoriotools on github.

I'm currently migrating the docker registry https://hub.docker.com/r/dtandersen/factorio to 
https://hub.docker.com/r/factoriotools/factorio , when this is finished ill update it to the latest tag.

While all this is ongoing, @dtandersen can you please migrate this repository to factoriotools/docker_factorio_server on github, i added you as owner too?
So when this is done i can, properly setup the travis config to build the tags.

For now, update this repository and pull the image from https://hub.docker.com/r/factoriotools/factorio .

**Comment by Fank** [comment]
@SuperSandro2000 what is your docker hub username?

**Comment by SuperSandro2000** [comment]
@Fank supersandro2000 https://hub.docker.com/u/supersandro2000

**Comment by dtandersen** [comment]
Goodbye 5 million download counter! :)

**Comment by dtandersen** [comment]
@fank I'm in factorio tools on dockerhub, but not github.

**Comment by SuperSandro2000** [comment]
@dtandersen I invited you

**Comment by Fank** [comment]
@dtandersen i invited you in both, check https://github.com/factoriotools . You should see your invitation on the page.

**Comment by Fank** [comment]
And @SuperSandro2000 don't worry its not the first time i do something like this, the only problem we have is that the docker hub registry is unable to move, so this needed to be created new ...

**Comment by SuperSandro2000** [comment]
We could create manifests that point to the new location 🤔

**Comment by SuperSandro2000** [comment]
@dtandersen Just a friendly reminder that the repo needs to be moved before we can activate travis.

**Comment by MonkWho** [comment]
I'm assuming the hub has not been updated because Travis is not running yet? Can someone please push a manual update?

**Comment by mgh87** [comment]
The new github registry (currently in beta) might also be a way to go to minimize the friction from multiple platforms.

https://github.com/features/package-registry

**Comment by Fank** [comment]
@dtandersen Thanks for moving

For now this issue can be closed, new maintenance options are available because of the organization.

@mgh87 And no we will stay at dockerhub, for now but we could add additional registries (create a new issue for it, if you want it)
